settings.py

```diff
@@ -879,8 +879,12 @@
 # using the local DEBUG value (which we don't have access to here yet)
 COMPRESS_ENABLED = lambda: not DEBUG and not UNIT_TESTING  # noqa: E731
 COMPRESS_OFFLINE = lambda: not DEBUG and not UNIT_TESTING  # noqa: E731
-COMPRESS_CSS_FILTERS = ['compressor.filters.css_default.CssAbsoluteFilter',
-'compressor.filters.cssmin.rCSSMinFilter']
+COMPRESS_FILTERS = {
+    'css': [
+        'compressor.filters.css_default.CssAbsoluteFilter',
+        'compressor.filters.cssmin.rCSSMinFilter',
+    ]
```

Comment:
I know I made this suggestion, but did we verify that leaving out the js key does give us the default compression, rather than turning it off entirely?

Reply:
as far as i understand this code https://github.com/django-compressor/django-compressor/blob/bf9803bcd0c57101c9e37d8f4a77d64e3449a39b/compressor/conf.py#L163-L165, only the part that you specify is overwritten.